lib/core/src/lib/app-config/app-config.service.spec.ts

```diff
@@ -111,12 +111,12 @@ describe('AppConfigService', () => {
         expect(appConfigService.get('application.name')).toEqual('custom name');
     });
 
-    it('should stream only the selected attribute changes when using select',  fakeAsync(() => {
+    it('should stream only the selected attribute changes when using select',  async () => {
         appConfigService.config.testProp = true;
-        appConfigService.select('testProp').subscribe((property) => {
-            expect(property).toBeTruthy();
-        });
-    }));
+        const propertyPromise = appConfigService.select('testProp').toPromise();
```

Comment:
can we try to use `lastValueFrom() / firstValueFrom()`?
[FE unit test best practices](https://alfresco.atlassian.net/wiki/spaces/TECH/pages/510919005/FE+unit+tests+best+practices#Adapting-a-test-written-with-done()-to-use-promises-instead)

Reply:
Currently, we're using RxJS v6.6.6 and those functions are supported from v7 onwards 😄